src/modules/storagefacility/TableComponent.js

```diff
@@ -625,4 +533,57 @@ const props = {
   emitSuccess
 };
 
-export default inject(data, commands, props)(StorageUnitsContainer);
+const processBarcode = (barCode, props) => {
```

Comment:
I have been thinking alot about how to cut it up into reusable functions. Youll have to give suggestions for a comment like this. How am i supposed to refactor it?